test/select_items_integration_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Searis AS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"context"
+	"testing"
+
+	clarify "github.com/clarify/clarify-go"
+	"github.com/clarify/clarify-go/fields"
+)
+
+func TestSelectItems(t *testing.T) {
+	ctx := context.Background()
+	creds := getCredentials(t)
+	client := creds.Client(ctx)
+	prefix := createPrefix()
+
+	a := TestArgs{
+		ctx:    ctx,
+		creds:  creds,
+		client: client,
+		prefix: prefix,
+	}
+
+	applyTestArgs(a, onlyError(insertDefault), onlyError(saveSignalDefault), onlyError(publishSignalDefault))
+
+	got, err := selectItemDefault(a)
+
+	if err != nil {
+		t.Errorf("%v", err)
+	} else {
+		jsonEncode(t, got)
+	}
+}
+
+func selectItemDefault(a TestArgs) (*clarify.SelectItemsResult, error) {
+	query := createAnnotationQuery(a.prefix)
+
+	return selectItem(a.ctx, a.client, a.prefix, query)
+}
+
+func selectItem(ctx context.Context, client *clarify.Client, prefix string, query fields.ResourceQuery) (*clarify.SelectItemsResult, error) {
+	result, err := client.Clarify().SelectItems(query).Do(ctx)
+
+	return result, err
+}
```

Comment:
We can probably drop this helper method; it is effectively possible to repeat as a one-liner.
```suggestion
func selectItems(ctx context.Context, client *clarify.Client, query fields.ResourceQuery) (*clarify.SelectItemsResult, error) {
	result, err := client.Clarify().SelectItems(query).Do(ctx)

	return result, err
}
```